docker/Dockerfile

```diff
@@ -0,0 +1,68 @@
+############################################
+# Dockerfile to run the PCIC data portal   #
+# Based on Ubuntu 16.04                    #
+############################################
+
+FROM ubuntu:16.04
+MAINTAINER Carl Masri <cmasri@uvic.ca>
+
+# Get the system level dependencies
+RUN apt-get update && apt-get install -y \
+    python-dev \
+    python-virtualenv \
+    build-essential \
+    libhdf5-dev \
+    libgdal-dev \
+    libnetcdf-dev \
+    git
```

Comment:
I think we also need to apt-get install libffi-dev, for requests[security] to install properly. (At least I get lots of warnings w/o it).